src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs

```diff
@@ -1722,20 +1722,29 @@ public string AppDomainName
             }
         }
 
+        /// <summary>
+        /// Gets or sets the PipeName to connect to.
```

Comment:
Should we mention "debug"?

Reply:
fixed this with rename to CustomPipeName